docs/source/onedal/algorithms/clustering/kmeans.rst

```diff
@@ -85,10 +85,16 @@ to each cluster.
 .. math::
    c_j^{(t + 1)} = \frac{1}{|S_j^{(t)}|} \sum_{x \in S_j^{(t)}} x, \quad 1 \leq j \leq k.
 
+If it turned out that some of :math:`S_j^{(t)}` are empty then start empty clusters handling procedure.
```

Comment:
```suggestion
If some of :math:`S_j^{(t)}` are empty, start the empty clusters handling procedure.
```

Reply:
Not clear what is meant by "some". If even one empty thing triggers the procedure, it is better to use "any". If it is more than one, we should clarify what the minimal condition is.

Reply:
done